src/main/java/com/cegeka/academy/service/invitation/InvitationService.java

```diff
@@ -21,4 +22,8 @@
 
     void sendGroupInvitationsToPrivateEvents(Long idGroup, Invitation invitation) throws NotFoundException;
 
+    Invitation createChallengeInvitationForOneUser (InvitationDTO invitationDTO, Long challengeId) throws ExistingItemException, NotFoundException;
+
+    Invitation getInvitation(Long invitationId);
```

Comment:
Am creat metoda getInvitation pentru a face o verificare suplimentara a ceea ce s-ar returna. Am observat ca obiectul de tip Invitation returnat are User=null. Am verificat si prin returnarea unui UserChallenge, care are in interiorul sau si un obiect de tip Invitation, iar acolo era totul in regula.